ffi/build.py

```diff
@@ -112,11 +112,11 @@ def main_posix(kind, library_ext):
     if not (out.startswith('8.0.') or out.startswith('7.0.')
             or out.startswith('7.1.')):
         msg = (
-            "Building llvmlite requires LLVM 7.0.x, 7.1.x or 8.0.x Be sure to "
-            "set LLVM_CONFIG to the right executable path.\n"
+            "Building llvmlite requires LLVM 7.0.x, 7.1.x or 8.0.x, got %r. "
+            "Be sure to set LLVM_CONFIG to the right executable path.\n"
             "Read the documentation at http://llvmlite.pydata.org/ for more "
             "information about building llvmlite.\n"
-            )
+            % (out.strip(),))
```

Comment:
Need this be a tuple?

Reply:
Also, perhaps use new style formatting?

Reply:
> Need this be a tuple?

No -- in general I find it more future compatible to make it one in case other things need to be added in the future, but I'll revert

> Also, perhaps use new style formatting?

Sure, I'll change it. I was trying to conform to the local file convention since `format` is unused and `%` used 5 times, but happy either way